docs/test-suite/ReFrame-configuration-file.md

```diff
@@ -313,13 +316,23 @@ The most common configuration items defined at this level are:
   _feature_ (for example if GPUs are available). Feature names are standardized in the EESSI test suite in
   [`eessi.testsuite.constants.FEATURES`](https://github.com/EESSI/test-suite/blob/main/eessi/testsuite/constants.py)
   dictionary.
-  Typically, you want to define `features: [FEATURES[CPU]]` for CPU based partitions, and `features: [FEATURES[GPU]]`
-  for GPU based partitions. The first tells the EESSI test suite that this partition can only run CPU-based tests,
-  whereas second indicates that this partition can only run GPU-based tests.
-  You _can_ define a single partition to have _both_ the CPU and GPU features (since `features` is a Python list).
+  Typically, you want to define `features: [FEATURES[CPU]] + list(SCALES.keys())` for CPU based partitions, and
+  `features: [FEATURES[GPU]] + list(SCALES.keys())` for GPU based partitions. The first tells the EESSI test suite
+  that this partition can only run CPU-based tests, whereas second indicates that this partition can only run GPU-based
+  tests. You _can_ define a single partition to have _both_ the CPU and GPU features (since `features` is a Python list).
   However, since the CPU-based tests will not ask your batch scheduler for GPU resources, this _may_ fail on batch
   systems that force you to ask for at least one GPU on GPU-based nodes. Also, running CPU-only code on a GPU node is
   typically considered bad practice, thus testing its functionality is typically not relevant.
+  The `list(SCALES.keys())` adds all the scales that may be used by EESSI tests to the `features` list. These scales are
+  defined in
+  [`eessi.testsuite.constants.SCALES`](https://github.com/EESSI/test-suite/blob/main/eessi/testsuite/constants.py)
+  and define at which scales tests should be run, e.g. single core, half a node, a full node, two nodes, etc. This
+  can be used to exclude running at certain scales on systems that would not support it. E.g. some systems might not
+  support requesting multiple partial nodes, which is what the `1_cpn_2_nodes` (1 core per node, on two nodes) and
+  `1_cpn_4_nodes` scales do. One could exclude these by setting e.g.
+  `features: [FEATURES[CPU]] + [s for s in SCALES if s not in ['1_cpn_2_nodes', '1_cpn_4_nodes']]`. With this
+  configuration setting, ReFrame will run all the scales listed in `eessi.testsuite.constants.SCALES _except_
+  those two. In a similar way, one could exclude all multinode tests if one just has a single node available.
```

Comment:
@casparvl Maybe we need one more sentence here to explain what will happen if `SCALES` is not used to define `features` (no tests will match, so nothing will get run, I think).
And we maybe should even print a warning in that case, if possible, and suggest to update your configuration file to ensure that at least one scale is listed in `features`?